usaspending_api/core/validator/award_filter.py

```diff
@@ -50,4 +52,8 @@
     a['optional'] = a.get('optional', True)  # future TODO: want to make time_period required
     a['key'] = 'filters{sep}{name}'.format(sep=TINY_SHIELD_SEPARATOR, name=a['name'])
     if a['type'] == 'array':
-        a['min'] = a.get('min', 1)
+        a['array_min'] = a.get('array_min', 1)
+        a['array_max'] = a.get('array_max', 0)
```

Comment:
Originally, a max value is applied to a model if it is not provided. It is probably a good idea to set an array and object max values. Even if the max is quite large (e.g. 5000) it is still providing an upper boundary for API users